src/Lykke.blue.Service.ReferralLinks.Core/Settings/ServiceSettings/Services.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Lykke.Service.Kyc.Client;
+
+namespace Lykke.blue.Service.ReferralLinks.Core.Settings.ServiceSettings
+{
+    public class Services
```

Comment:
It is preferable to use existing client settings sections. Take a look at "BalancesServiceClient" section. You need to extend access token (like BalancesServiceClient.*) for you service. This way we use single point of configuring service client settings.